metagpt/document_store/qdrant_store.py

```diff
@@ -0,0 +1,90 @@
+from dataclasses import dataclass
+from typing import List
+
+from qdrant_client import QdrantClient
+from qdrant_client.http.models import VectorParams
+from qdrant_client.models import Filter, PointStruct
+
+from metagpt.document_store.base_store import BaseStore
+
+
+@dataclass
+class QdrantConnection:
+    url: str = None
+    host: str = None
+    port: int = None
+    memory: bool = False
+    api_key: str = None
+
+
+class QdrantStore(BaseStore):
```

Comment:
How to extend the usage of QdrantStore in memory, maybe you can make a supplement.

Reply:
QdrantConnection memory is True，it will use qdrant in meory

Reply:
The usage of QdrantStore which I mentioned is where to setup QdrantConnection in `key.yaml` or using Env methods.